pkg/registry_checker/checker.go

```diff
@@ -193,7 +194,17 @@ func NewRegistryChecker(
 	}
 	rc.controllerIndexers.cronJobIndexer = rc.cronJobsInformer.Informer().GetIndexer()
 
-	rc.controllerIndexers.secretIndexer = rc.secretsInformer.Informer().GetIndexer()
+	namespace := "default"
+	// Create a context
+	ctx := context.TODO()
```

Comment:
Let’s pass the background context.

Reply:
Sure, could you give me a hint, what a good way to do this would be? Unfortunately i am no go expert.
Can i extract the context from here: https://github.com/deckhouse/k8s-image-availability-exporter/blob/master/pkg/registry/indexers.go#L288 later on, or which background context are you referring to? I guess the indexers don't get an additional context passed to them?
Thanks!